src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -113,25 +114,6 @@ public string NamedUser
             }
         }
 
-        private EventHandler<DeepLinkEventArgs> onDeepLinkReceived;
```

Comment:
This section (same in iOS) was causing crashes. EventHandler is a delegate type, which is essentially a function pointer. If you declare one like this as an explicit backing delegate for an event, it will be initially as null, just like any other non-primitive variable. This is not C++, where declaring a variable invokes its class' default constructor. It's not that you can't do this, but rather that you have to check for null when using += to add listeners to it, such as below. I don't think this pattern of lazily adding native listeners buys us a whole lot, though, so I changed to be similar to the way we're dealing with Message Center events by adding listeners/NSNotification observers after takeOff.

Reply:
I misunderstood the source of the crash. Disregard the above.